pages/common/scala.md

```diff
@@ -7,6 +7,10 @@
 
 `scala`
 
+- Start the interpreter with a fat jar in the classpath:
```

Comment:
What does 'fat' mean here? :smile:

Reply:
I meant a jar with all its dependencies (other jars) inside it. But to be honest, I think "dependency" is a better word for that. I'll push an update. 😅